docs/cli.md

```diff
@@ -33,7 +33,7 @@ The encrypted private key can be obtained from the `.octez-client/` directory
 
 ## Importing the generated key using signatory-cli
 
-A private key can be imported into any of the backend vaults (except: AWS,file & ledger) using the below command.
+A private key can be imported into any of the backend vaults (except: AWS & ledger) using the below command.
```

Comment:
Is the signatory supporting import for the file vault now?

Reply:
file vault not included

Reply:
File vault should be there in the `except:` list

Reply:
File vault should be included. please change this.